pesign-obs-integration.spec

```diff
@@ -31,15 +31,15 @@ Requires:       pesign
 %endif
 BuildRequires:  openssl
 Url:            http://en.opensuse.org/openSUSE:UEFI_Image_File_Sign_Tools
-Source:         %{name}_%{version}.tar.gz
-BuildRoot:      %{_tmppath}/%{name}-build
+Source:         %{name}-%{version}.tar.xz
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
```

Comment:
The reason that we use "%{name}_%{version}.tar.gz" and "%{_tmppath}/%{name}-build" is to make the tarball support both openSUSE and Debian at the same time, and this might break the debian build.

Reply:
I didn't know that (is it documented anywhere?). The Debian interoperatbility is a good thing, but really not something I want to dig into. You guys know your ways around that.

Maybe you could simply integrate modsign-verify yourself?

Reply:
I was hoping to be able to use a _service file to simplify package creation. But OBS tar-scm.service supports only "-" to separate name and version. Would it work for Debian not to include the version at all? The tarball name itself doesn't matter, but the path of the topdir in the tarball (-n arg to %setup) does.

    Source: %{name}.tar
    Buildroot: ${_tmppath}/%{name}-build
    %setup -D -n %{name}

Btw does the buildroot really matter for Debian?

Reply:
OK, @lcp keep me in the loop please. I'd like to double check the integration.